lib/glific/third_party/gcs/gcs.ex

```diff
@@ -32,30 +32,39 @@ defmodule Glific.GCS do
   @doc """
   Creating a dataset with messages and contacts as tables
   """
-  @spec refresh_gcs_setup(non_neg_integer) :: :ok
+  @spec refresh_gcs_setup(non_neg_integer) :: {:ok, any} | {:error, any}
   def refresh_gcs_setup(organization_id) do
     Logger.info("refresh GCS setup for org_id: #{organization_id}")
 
-    organization_id
-    |> insert_gcs_jobs()
+    case insert_gcs_jobs(organization_id) do
+      {:ok, gcs} -> {:ok, gcs}
+      {:error, error} -> {:error, error}
+    end
 
-    :ok
+    # organization_id
+    # |> insert_gcs_jobs()
   end
 
   @doc false
-  @spec insert_gcs_jobs(non_neg_integer) :: :ok
+  @spec insert_gcs_jobs(non_neg_integer) :: {:ok, any} | {:error, any}
   def insert_gcs_jobs(organization_id) do
     Repo.fetch_by(GcsJob, %{organization_id: organization_id})
     |> case do
       {:ok, gcs_job} ->
-        gcs_job
+        {:ok, gcs_job}
 
       _ ->
-        %GcsJob{organization_id: organization_id}
-        |> Repo.insert!()
-    end
+        case %GcsJob{} |> GcsJob.changeset(organization_id) |> Repo.insert() do
+          {:ok, organization} ->
+            {:ok, organization}
 
-    :ok
+          {:error, error} ->
+            {:error, error}
+        end
```

Comment:
can you revert these changes and create a separate PR stating what problem it is solving

Reply:
It is being used by partner.ex file. Look at the function there credential_update_callback. It expect tuple instead of just :ok

Reply:
Here also as we are returning the same tuple so adding case statement does not make sense